src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -102,7 +102,12 @@ public partial class CultureInfo : IFormatProvider, ICloneable
         private static volatile CultureInfo? s_userDefaultUICulture;
 
         // The Invariant culture;
+// https://github.com/dotnet/runtime/issues/94225
+#if TARGET_BROWSER
```

Comment:
Is this problem for all Mono+AOT or just for wasm ? What about wasi ?

Reply:
Wasi does not support HybridGlobalization, so no problem there. @mkhamoyan, could you please check if HG + AOT throw in `System.Globalization.Hybrid.WASM.Tests.csproj` on iOS?

Reply:
I tried running on `iOS` with `./dotnet.sh build src/libraries/System.Runtime/tests/System.Globalization.Tests/Hybrid/System.Globalization.Hybrid.WASM.Tests.csproj /t:Test /p:TargetOS=maccatalyst /p:TargetArchitecture=x64 /p:Configuration=Debug /p:RunAOTCompilation=true /p:EnableAggressiveTrimming=true /p:UseSystemResourceKeys=true` but it doesn't crash. Can you share the whole command for wasm that `System.Globalization.Hybrid.WASM.Tests.csproj`crashes?

Reply:
`dotnet build -bl /t:Test src/libraries/System.Globalization/tests/System.HybridGlobalization.Tests.csproj /p:Configuration=Debug /p:TargetOS=browser /p:EnableAgressiveTrimming=true /p:RunAOTCompilation=true`